Gemfile

```diff
@@ -15,8 +15,7 @@ end
 # only the master branch is supported on rspec-support
 gem "rspec-support", :git => "git://github.com/rspec/rspec-support.git"
 
-gem "cucumber", "~> 1.1.9"
-gem "aruba",    "~> 0.5"
+gem "aruba"
```

Comment:
Yeah I tried, however the older version wasn't working on JRuby, and this was an "easier" fix, I could pin it to the version in use?